Bundle/WidgetBundle/Builder/WidgetFormBuilder.php

```diff
@@ -136,7 +136,7 @@ public function renderNewQuantumForms($slot, View $view, $widgets, $activeWidget
     {
         $forms = [];
         foreach ($widgets as $key => $widget) {
-            $_quantum = null !== $quantum ? $quantum : $key;
+            $_quantum = null !== $quantum ? $quantum : $this->convertToString($key);
```

Comment:
I can't remember why the string 'static' was given, I agree with you but please be careful doing so